### PR TITLE
feat(quantic): new component the quanticUserActionSession created

### PR DESCRIPTION
**Pull request by mmitiche**
[SFINT-4469](https://coveord.atlassian.net/browse/SFINT-4469)
### Demo:

https://user-images.githubusercontent.com/86681870/210596693-45bb2a85-7214-475e-b882-b1c58f4d3893.mov

### Mockups:
https://www.figma.com/file/tc1NDbxceu3tW1UjFIt8cm/Quantic-Insight-Panel?node-id=204%3A13521&t=ZDDS6GPff8JVxK5P-0
Mockups

[SFINT-4469]: https://coveord.atlassian.net/browse/SFINT-4469?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SFINT-4469]: https://coveord.atlassian.net/browse/SFINT-4469?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/quantic/force-app/main/default/lwc/quanticUserActionSession/__tests__/quanticUserActionSession.test.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+      await flushPromises();
+
+      const userActionEvents  = element.shadowRoot.querySelectorAll(
+        selectors.userActionEvent
+      );
+      const moreActionsButton = element.shadowRoot.querySelector(
+        selectors.moreActionsButton
+      );
+
+      userActionEvents .forEach((userActionEvent, index) => {
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/lwc/quanticUserActionSession/__tests__/quanticUserActionSession.test.js:139]:
```suggestion
      userActionEvents.forEach((userActionEvent, index) => {
```

**Review by lbergeron — APPROVED** [review]
Well done!

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 2]
```diff
@@ -903,4 +903,25 @@
       <protected>false</protected>
       <shortDescription>Copied!</shortDescription>
     </labels>
+    <labels>
```
Comment (mmitiche) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:906]: @acarpentier01 Let me know if you think there is anything we can improve in the labels defined in this file. Thank you!

File: packages/quantic/force-app/main/translations/fr.translation-meta.xml [hunk 3]
```diff
@@ -476,4 +476,16 @@
       <label>Copié!</label>
       <name>quantic_Copied</name>
     </customLabels>
+    <customLabels>
```
Comment (mmitiche) [inline packages/quantic/force-app/main/translations/fr.translation-meta.xml:479]: @acarpentier01 the translations are here as well, all your suggestions are very welcome!

**Review by SimonMilord — APPROVED** [review]

**Review by acarpentier01 — APPROVED** [review]
LGTM! Just one minor issue.

**Comment by mmitiche** [comment]
Will be back to this when we will be ready for the user actions.

**Pull request closed without merging**